bugz/cli_argparser.py

```diff
@@ -260,6 +260,8 @@ def make_arg_parser():
                              help='load description from file')
     post_parser.add_argument('--append-command',
                              help='append output from command to description')
+    post_parser.add_argument('--package-list',
+                             help='stabilization package list')
```

Comment:
Please add some marking to show it is Gentoo support feature only (in the help). Other bugzilla instances don't support it.